NitroxClient/Communication/Packets/Processors/PDAScannerEntryProcessor.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection;
+using NitroxClient.Communication.Abstract;
+using NitroxClient.Communication.Packets.Processors.Abstract;
+using NitroxClient.GameLogic.Helper;
+using NitroxModel.DataStructures.Util;
+using NitroxModel.Logger;
+using NitroxModel.Packets;
+using UnityEngine;
+
+namespace NitroxClient.Communication.Packets.Processors
+{
+    public class PDAScannerEntryProcessorAdd : ClientPacketProcessor<PDAEntryAdd>
```

Comment:
Can you please tweak the name of the classes in here to match our format: <PacketName>Processor.  Also, please have separate files for each processor class.